components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2Service.java

```diff
@@ -676,7 +678,9 @@ public OAuthRevocationResponseDTO revokeTokenByOAuthClient(OAuthRevocationReques
 
                 } else if (accessTokenDO != null) {
                     if (revokeRequestDTO.getConsumerKey().equals(accessTokenDO.getConsumerKey())) {
-                        if ((OAuth2Util.getAppInformationByClientId(accessTokenDO.getConsumerKey()).
+                        // Extracting the application details with consumer key and tenant domain.
+                        String tenantDomain = IdentityTenantUtil.getTenantDomain(accessTokenDO.getTenantID());
+                        if ((OAuth2Util.getAppInformationByClientId(accessTokenDO.getConsumerKey(), tenantDomain).
```

Comment:
Will the same concern we discussed in https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/302 applicable here also?

Reply:
Since this getting from `accessTokenDO.getTenantID()`, this issue is not there